pkg/notes/notes.go

```diff
@@ -422,9 +422,9 @@ func (g *Gatherer) ReleaseNoteFromCommit(result *Result) (*ReleaseNote, error) {
 	documentation := DocumentationFromString(prBody)
 
 	author := pr.GetUser().GetLogin()
-	authorURL := fmt.Sprintf("https://github.com/%s", author)
+	authorURL := fmt.Sprintf("%s/%s", g.options.GithubBaseURL, author)
```

Comment:
Perhaps changing this assignment like this could do the trick?
```suggestion
	authorURL := author := pr.GetUser().GetURL()
```
And perhaps the same for the PR, sometimes these methods are empty but perhaps they work

Reply:
Oh, that's nice! I have tried that. Just one thing, instead of `GetURL()`(which gives an API url), it should be `GetHTMLURL()`.